### PR TITLE
Youtube design implemented

### PR DESCRIPTION
**Pull request by mbilal-dev9**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  display: flex;
+  align-items: center;
+  background-color: #272727;
+  height: 40px;
+  width: 90%;
+  border-radius: 20px;
+}
+
+.input-field {
+  width: 90%;
```
Comment (umarabdullah23) [inline css/index.css]: use 
calc(100% - 50px)
so that search icon width is fixed

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  justify-content: center;
+  align-items: center;
+}
+
+.profileIcon {
+  cursor: pointer;
+}
+
+.main-sidebar-div {
+  top: 54px;
```
Comment (umarabdullah23) [inline css/index.css:125]: display: flex;
    flex-direction: column;
    row-gap: 20px;

File: css/index.scss [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+.main-sidebar-div {
+  top: 54px;
+  padding: 0;
+  width: 100px;
+  height: calc(100vh - 54px);
+  background-color: black;
+  position: fixed;
+
+  .aside-icons {
```
Comment (umarabdullah23) [inline css/index.scss:147]: margin-left: auto;
    margin-right: auto;

to horizontally center items of sidebar

File: css/index.css [hunk 4]
```diff
@@ -0,0 +1,165 @@
+body {
```
Comment (umarabdullah23) [inline css/index.css:1]: Add scrollbar on main content

File: css/index.css (hunk 1 above)
Comment (mbilal-dev9) [reply css/index.css]: Done.

File: css/index.css (hunk 2 above)
Comment (mbilal-dev9) [reply css/index.css:125]: Done.

File: css/index.scss (hunk 3 above)
Comment (mbilal-dev9) [reply css/index.scss:147]: Done.

**Pull request merged**